package/crds/kafka.aws.crossplane.io_clusters.yaml

```diff
@@ -479,6 +482,7 @@ spec:
                     description: Create tags when creating the cluster.
                     type: object
                 required:
+                - clusterName
```

Comment:
would defaulting this parameter to the object name provide backwards compatibility? Would it make sense?

Reply:
problem is for people using compositions - the cr.name is complete different